features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -768,6 +774,12 @@
     <bundle start-level="80">mvn:org.openhab.io/org.openhab.io.gpio/${project.version}</bundle>
   </feature>
 
+  <feature name="openhab-io-gcal1" description="GCal" version="${project.version}">
+    <feature>openhab-runtime-base</feature>
+    <feature>openhab-runtime-compat1x</feature>
+    <bundle start-level="80">mvn:org.openhab.io/org.openhab.io.gcal/${project.version}</bundle>
+  </feature>
```

Comment:
The binding itself actually doesn't know where its configuration comes from.  The 1.x runtime used openhab.cfg, but it's refactored in OH2 to have a .cfg per service, gcal.cfg in this case, for text file configuration.  Or via Paper UI so no config file has to be used.  

To add gcal.cfg, just take the GCal section from openhab_default.cfg into features/openhab-addons-external/src/main/resources/conf/gcal.cfg (but remove the `gcal:` prefixes in it).  Modify `features/openhab-addons-external/pom.xml` like in #4814.

To allow GCal to be configured from the Paper UI, add bundles/io/org.openhab.io.gcal/ESH-INF/config/config.xml similar to #4049, and see [this documentation](https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#configuration-descriptions).